src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -0,0 +1,666 @@
+                      }).when(mockObjBar).getMsg();
+                      doAnswer(invocation -> {
+                          String echo = invocation.getArgument(0);
+                          return "BAR" + echo;
+                      }).when(mockObjBar).getMsg(nullable(String.class));
+                      try (MockedConstruction mockConsFoo = mockConstructionWithAnswer(Foo.class, delegatesTo(mockObjFoo));MockedConstruction mockConsBar = mockConstructionWithAnswer(Bar.class, delegatesTo(mockObjBar))) {
+                          assertEquals("FOO", new Foo().getMsg());
+                          assertEquals("FOOecho", new Foo().getMsg("echo"));
+                          assertEquals("BAR", new Bar().getMsg());
+                          assertEquals("BARecho", new Bar().getMsg("echo"));
+                      }
+                  }
+              }
+              """)
+        );
+    }
+
+    @Test
+    void mockUpInnerStatementTest() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import mockit.Mock;
+              import mockit.MockUp;
+              
+              import org.junit.Test;
+              import static org.junit.Assert.assertEquals;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      new MockUp<MyClazz>() {
+                          final String msg = "newMsg";
+              
+                          @Mock
+                          public String getMsg() {
+                              return msg;
+                          }
+                      };
+              
+                      // Should ignore the newClass statement
+                      new Runnable() {
+                          @Override
+                          public void run() {
+                              System.out.println("run");
+                          }
+                      };
+                      assertEquals("newMsg", new MyClazz().getMsg());
+                  }
+              
+                  public static class MyClazz {
+                      public String getMsg() {
+                          return "msg";
+                      }
+                  }
+              }
+              """, """
+              import org.junit.Test;
+              import org.mockito.MockedConstruction;
+              
+              import static org.junit.Assert.assertEquals;
+              import static org.mockito.AdditionalAnswers.delegatesTo;
+              import static org.mockito.Answers.CALLS_REAL_METHODS;
+              import static org.mockito.Mockito.*;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      final String msg = "newMsg";
+                      MockUpTest.MyClazz mockObjMockUpTestMyClazz = mock(MockUpTest.MyClazz.class, withSettings().defaultAnswer(CALLS_REAL_METHODS));
```

Comment:
Would be good to rename this var to mockMyClazz and also remove the FQN, as uneeded as MyClass is an inner class here of this test